pkgs/by-name/sd/sdl3/package.nix

```diff
@@ -160,7 +143,21 @@ stdenv.mkDerivation (finalAttrs: {
       vulkan-loader
     ]
     ++ lib.optional (openglSupport && !stdenv.hostPlatform.isDarwin) libGL
-    ++ lib.optional x11Support xorg.libX11;
+    ++ lib.optional x11Support xorg.libX11
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # error: 'MTLPixelFormatASTC_4x4_LDR' is unavailable: not available on macOS
+      (darwinMinVersionHook "11.0")
+
+      apple-sdk_11
+    ]
+    ++ lib.optionals ibusSupport [
+      # sdl3 only uses some constants of the ibus headers
+      # it never actually loads the library
+      # thus, it also does not have to care about gtk integration,
+      # so using ibusMinimal avoids an unnecessarily large closure here.
+      ibusMinimal
+    ]
+    ++ lib.optional waylandSupport zenity;
```

Comment:
I don't think zenity is linked to? I think it's invoked as a command

Reply:
indeed, good catch: https://github.com/libsdl-org/SDL/blob/d14cbd7b509c986708e03708096015c4023099f6/src/dialog/unix/SDL_zenitydialog.c#L350
And we even patch sdl to use an absolute path for that.